vaadin-components-gwt/src/main/java/com/vaadin/components/grid/GridComponent.java

```diff
@@ -288,14 +288,18 @@ public IndexBasedSelectionModel getSelectionModel() {
         return (IndexBasedSelectionModel) grid.getSelectionModel();
     }
 
-    public void refresh() {
+    public void forceRefresh() {
```

Comment:
This API feels redundant. refresh(boolean) is still used directly when the value is false, otherwise this is used??? Why not just refresh(true)?